blog/templates/blog/post_detail.html

```diff
@@ -1,16 +1,23 @@
 {% extends 'blog/base.html' %}
 
+{% load i18n %}
+
 {% block content %}
   <div class="post">
     {% if post.published_date %}
       <div class="date">
         {{ post.published_date }}
       </div>
+    {% else %}
+      <a class="btn btn-default" href="{% url 'post_publish' pk=post.pk %}">{% trans "Publish" %}</a>
```

Comment:
Will unauthenticated dude see this?

Reply:
Unauthenticated dude cant goes to page with drafts of posts, then he doesn't see unpublished posts.
He can only manually writes link and goes to page with unpublished post, but i'll fix this later

Reply:
I mean, is this link shown in their interface?